### PR TITLE
Ispfsample

### PR DESCRIPTION
**Pull request by MikeFultonDev**

### CODE REVIEW

**Review by AKippins — COMMENTED** [review]
This looks good. The only thing I'd maybe do is put Note: in Italics. So it'd be _Note:_ but it's not really an issue it's just something that may catch the eye a little better since its in a lot of text.

File: samples/one-liners.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ opercmd 'd u,dasd,online,,65536'
 ```
 
 Print all files on all online volumes:
- -Write out the online volumes to stdout, then strip off the 5 header lines and the one trailer line, using awk to print just the volume serial name and store it into the variable _volumes_.
- -For each volume in volumes, print out the volume table of contents if there is no error getting the VTOC for the volume. This will list cataloged and uncataloged datasets. 
- -Note: You could add a _grep_ or _awk_ filter on the end to restrict the output to files of a particular pattern.
+* Write out the online volumes to stdout, then strip off the 5 header lines and the one trailer line, using awk to print just the volume serial name and store it into the variable _volumes_.
+* For each volume in volumes, print out the volume table of contents if there is no error getting the VTOC for the volume. This will list cataloged and uncataloged datasets. 
+* Note: You could add a _grep_ or _awk_ filter on the end to restrict the output to files of a particular pattern.
```
Comment (AKippins) [inline samples/one-liners.md:28]: Maybe _Note:_ or **Note:** so that we catch the eye a little better.

**Comment by govarney** [comment]
Looks good.  My only suggestion is possibly add an extra line in the help, stating that the HLQ for the ISPF supplied datasets can be setup via ISPF_HLQ.   The overwhelming major of systems I work on don't use ISP, so I'd find this useful.

**Comment by MikeFultonDev** [comment]
> This looks good. The only thing I'd maybe do is put Note: in Italics. So it'd be _Note:_ but it's not really an issue it's just something that may catch the eye a little better since its in a lot of text.

Great point - I'll do that

**Comment by MikeFultonDev** [comment]
> Looks good. My only suggestion is possibly add an extra line in the help, stating that the HLQ for the ISPF supplied datasets can be setup via ISPF_HLQ. The overwhelming major of systems I work on don't use ISP, so I'd find this useful.

Will do - thanks for catching that. Is there a better default you can recommend?

**Pull request merged**